### DISCUSSION TITLE
Fails to create vector store.

### DISCUSSION THREAD

**Discussion by JustAStudentOfTechnology** [logancyang/obsidian-copilot#561] [Q&A]
ISSUE:
AN ERROR OCCURED SAVING VAULT TO VECTOR STORE

Dev Console Logs:
plugin:copilot:87066 OpenAI API key is not provided for the embedding model.

plugin:copilot:111904 Error saving vault to vector store: Error: Embedding instance not found.
    at CopilotPlugin.indexVaultToVectorStore (plugin:copilot:111934:13)
    at CopilotPlugin.onload (plugin:copilot:111902:20)

This is a fresh install with the following items:
LM Studio Current Version: 0.2.27
Model:"QuantFactory/Meta-Llama-3-8B-Instruct-GGUF"
GPU Offload: Max
Context Length: 8192 (Model Max)
Server Model: Default LM Studio Windows
CORS : On
Obsidian Installer Version: 1.5.3
Obsidian Current Version: 1.6.5
Copilot: LM Studio Local
Auto Index: On Startup

EXPECTED BEHAVIOR
It is supposed to complete an index of the vault which is approximately 100 notes large. Does not even get to 0/100.

**Comment by SyndicatedPillbug** [comment]
Same error

**Comment by logancyang** [comment]
What embedding model did you pick? Can you share a screenshot of your setting.
